playbook-website/app/javascript/components/Website/src/layouts/Sidebar/NavComponents/KitsNavComponent.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect, useContext } from "react"
+import { linkFormat } from "../../../../../../utilities/website_sidebar_helper"
```

Comment:
This path is a little rough and I wonder if a Webpack alias is a better fit. Relative paths are hard to work with.

Reply:
Made this much better. I had copied many of these files over in prep for upcoming work. 
Went ahead and created a hook, and co-located it much closer to this file, so the import doesn't look too too crazy now.